config/servers/web.js

```diff
@@ -110,11 +110,7 @@ exports.production = {
   servers: {
     web: (api) => {
       return {
-        padding: null,
-        metadataOptions: {
-          serverInformation: false,
```

Comment:
Shouldnt it be also set to false in production? Why remove it?

Reply:
when set it as default this also applies to any other environment if you dont modify it :)